### DISCUSSION TITLE
Borrowing Documents

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#207] [Q&A] [answered]
Currently, to borrow, you select from the main window 

- right button :  Borrow selected item... it opens the member manager

on the top menu
- 1 icon representing a home folder; it opens the members manager

on the top menu next to it
- 1 icon with 2 heads : it opens the members manager

can you explain me the difference between the 2 actions with the 2 icons?

**Comment by textbrowser** [comment, marked as answer]
Same functions for now. Separate functionality someday.

**Comment by textbrowser** [comment]
The separate functions are intended for separate perspectives. The Members Browser is not solely used for borrowing. It has other purposes. There will also be some improvements added to the reservation processes. For example, batches of items.

**Reply by meteos77** [reply]
this seems a good horizon
